ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py

```diff
@@ -46,6 +47,7 @@ def get_jobs(yaml_file, mode='y'):
         'fips-check',
```

Comment:
hi @dis016 Could you please also add a keyword `freeipa` here? we don't need to cover this kind of upgrade jobs.

Here is the description:
https://github.com/openshift/release/pull/54740#discussion_r1690925295